Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -16,23 +17,60 @@ public class ReportService : IReportService
         public void UpdateDashboardData(long instanceId)
         {
             UpdateAtlassianUsers(instanceId);
+            UpdateUsersWorklogs(instanceId);
         }
         #endregion
 
         #region Helpers
+        private void UpdateUsersWorklogs(long instanceId)
+        {
+            var worklogs = GetAtlassianWorklogs(instanceId);
+
+            using (var db = new ReportsDb())
+            {
+                var dbWorklogs = db.AtlassianWorklog
+                    .Where(p => p.InstalledInstanceId == instanceId)
+                    .ToList();
+
+                foreach (var worklog in worklogs)
```

Comment:
You should also handle the case when the worklog was deleted in JIRA.

Reply:
Done.